src/components/Send.jsx

```diff
@@ -1,20 +1,162 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { useState } from 'react'
 import { useLocation } from 'react-router-dom'
 import * as rb from 'react-bootstrap'
-import { serialize, ACCOUNTS } from '../utils'
+import PageTitle from './PageTitle'
+import ToggleSwitch from './ToggleSwitch'
+import { useCurrentWalletInfo, useSetCurrentWalletInfo, useCurrentWallet } from '../context/WalletContext'
+import { useSettings } from '../context/SettingsContext'
 import * as Api from '../libs/JmWalletApi'
 
-export default function Payment({ currentWallet }) {
+const isValidAddress = (candidate) => {
+  return typeof candidate === 'string' && !(candidate === '')
+}
+
+const isValidAccount = (candidate) => {
+  const parsed = parseInt(candidate, 10)
+  return !isNaN(parsed) && parsed >= 0
+}
+
+const isValidAmount = (candidate) => {
+  const parsed = parseInt(candidate, 10)
+  return !isNaN(parsed) && parsed > 0
+}
+
+const isValidNumCollaborators = (candidate) => {
+  const parsed = parseInt(candidate, 10)
+  return !isNaN(parsed) && parsed >= 1 && parsed <= 99
+}
```

Comment:
These aren't very strong validation rules. They're basically the same as before just extracted from the built-in form validation to give us a bit more fine grained control. We can update those in the future to e.g. only allow valid looking addresses etc.